server/package.json

```diff
@@ -1,17 +1,31 @@
 {
   "scripts": {
-    "dev": "ts-node-dev --respawn --transpile-only src/index.ts"
+    "dev": "nodemon -x ts-node src/index.ts",
+    "lint": "eslint --ext ts ./src",
+    "lint:fix": "yarn lint --fix",
+    "prettier": "prettier --ignore-path .gitignore --write src"
```

Comment:
You can run the following to make sure your code is up to standard

1. `npm run prettier`
2. `npm run lint:fix`